amp-story/templates/ampmars/json/bookend.json

```diff
@@ -0,0 +1,40 @@
+{
+    "bookendVersion": "v1.0",
+    "shareProviders": ["facebook", "twitter", "email"],
+    "components": [
+        {
+            "type": "heading",
+            "text": "Explore our other trips"
+        },
+        {
+            "type": "landscape",
+            "title": "See Venus up close",
+            "category": "Morning Star Fly-By",
+            "url": "https://www.example.com",
+            "image": "./img/07/morning-star-fly-by.jpg"
+        },
+        {
+            "type": "landscape",
+            "title": "A romantic getaway to Earth’s Moon",
+            "category": "Perfect Honeymoon",
+            "url": "https://www.example.com",
+            "image": "./img/07/perfect-honeymoon.jpg"
+        },
+        {
+            "type": "landscape",
+            "title": "A beautiful view from near space",
+            "category": "The Epic Sunset",
+            "url": "https://www.example.com",
+            "image": "./img/07/epic-sunset.jpg"
+        },
+        {
+            "type": "cta-link",
+            "links": [
+              {
+                "text": "Learn more",
+                "url": "example.com/signup"
```

Comment:
nit: preface the URL here with "https://www." to match the others.

Reply:
Fixed!